core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -664,13 +669,10 @@
       }
 
       .conversation-skin-supplemental-card-container {
-        left: 50%;
```

Comment:
Hi @maitbayev I think this works with `transform: translate(-50%)` an old way of centering the card.. before we started using flex. but if you see any issue introduced feel free to create one and I will try to fix. Thanks